sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -130,9 +130,9 @@ The Text Analytics client library provides a [TextAnalyticsClient][text_analytic
 asynchronous operations to access a specific use of Text Analytics, such as language detection or key phrase extraction.
```

Comment:
There are more references to `text input` in this document. Please modify

Reply:
updated